test/controllers/teams_users_controller_test.rb

```diff
@@ -0,0 +1,71 @@
+require 'test_helper'
```

Comment:
Could you tell us how to run those tests please? We are expecting test cases which can be run ban Travic CI.